### PR TITLE
Make app independently of the value of USE_TZ, pep8 correctness, doc, Fixed tests for Django 1.9

### PR DESCRIPTION
**Pull request by felixxm**
Make app independently of the value of USE_TZ

### CODE REVIEW

**Comment by felixxm** [comment]
Without this change you will get:

*RuntimeWarning: DateTimeField Request.time received a naive datetime (...) while time zone support is active.*

every time when TZ is set in settings.

File: docs/index.txt [hunk 1]
```diff
@@ -42,9 +42,9 @@ This command can be used to purge old requests, it takes two arguments amount an
 
 .. code-block:: bash
 
-    $ python manage.py purgerequests 1 month
+    $ python manage.py purgerequests 1 months
```
Comment (kylef) [inline docs/index.txt]: This should be as it was "1 month" not "1 months" since 1 is not plural.

**Comment by kylef** [comment]
Thanks for the pull request @felixxm, one minor comment but other than that this is great :+1:

**Comment by felixxm** [comment]
I also thought about changing *timedelta* to "relativedelta" (from python-dateutil) to get real month and year period.

File: docs/index.txt (hunk 1 above)
Comment (felixxm) [inline docs/index.txt]: Fixed

**Comment by felixxm** [comment]
Fixed tests for Django 1.9.

**Comment by kylef** [comment]
Thanks @felixxm, sorry for delay in merging this. There are no notifications when commits get added to a pull request so I'd suggest writing a comment once PR feedback has been resolved in the future.

**Pull request merged**